synapse/federation/federation_server.py

```diff
@@ -234,24 +233,17 @@ def __init__(self, hs):
                 continue
 
             try:
-                room_version = await self.store.get_room_version_id(room_id)
+                room_version = await self.store.get_room_version(room_id)
             except NotFoundError:
                 logger.info("Ignoring PDU for unknown room_id: %s", room_id)
                 continue
-
-            try:
-                format_ver = room_version_to_event_format(room_version)
-            except UnsupportedRoomVersionError:
+            except UnsupportedRoomVersionError as e:
                 # this can happen if support for a given room version is withdrawn,
                 # so that we still get events for said room.
-                logger.info(
-                    "Ignoring PDU for room %s with unknown version %s",
-                    room_id,
-                    room_version,
-                )
```

Comment:
If we're logging it seems useful to include *why* we're ignoring an event?

Reply:
yes, but `str(e)` will tell us that.

Reply:
But it won't log the room_id or the room version? (`UnsupportedRoomVersionError` doesn't include those things)

Reply:
https://github.com/matrix-org/synapse/blob/rav/redact_changes/6/synapse/storage/data_stores/main/state.py#L81-L82

Reply:
Huh, ok then. (The uses I have seen don't specify a custom message)